src/actions/index.js

```diff
@@ -1,10 +1,10 @@
-import dataNbaNet from 'apis/dataNbaNet';
+import dataNbaNet from 'apis';
 import { TEAMS, TEAM_COLORS } from 'enums';
 
 const resetPlayers = new Array(20).fill({});
 
 export const getTeams = (pathname, history) => async (dispatch) => {
-  dispatch({ type: 'GET_PLAYERS', payload: resetPlayers });
+  dispatch({ type: 'SET_PLAYERS', payload: resetPlayers });
```

Comment:
Due to the refactor comment we can now remove this line as our `playersListReducer` is being initialized with the empty arrays.

```
const initialPlayers = new Array(20).fill({});

export default (state = initialPlayers, action) => {
```